src/controllers/commands/commandMap.js

```diff
@@ -0,0 +1,7 @@
+import help from "./help.command.js";
+import checkin from "./checkin.command.js";
+
+export default {
+  help,
+  helthtoday: checkin, // key of the export is the command name: ie the user will call /helthtoday
```

Comment:
trivial: user sends command not in our list. what if we want to send /help to let user know command is not in the list and here's a list of commands you can use?

Reply:
we can try to figure out a way to do this if possible

Reply:
see #27